### ISSUE TITLE
Cursor jumps around and spaces added

### ISSUE THREAD

**Issue by bernini** [MustWin/irregardless-chrome#61]
I was able to repro both of the bugs Charles took a video of here. Cursor moves around a bunch and also spaces get added after "remove formatting," maybe when the service checks for tips again?

From Charles:
> Hey, here's a video where my cursor jumps to funny places without me actually clicking.  At :08 seconds, you'll see me remove a break between "Thanks," and "Charles" and then all of a sudden the space reappears, including extra space above.   Then starting at :24, you'll see my trying to remove line breaks and having a hard time.

https://drive.google.com/a/irregardless.ly/file/d/1ZkNZ50pu0sq6_lHaSzTrZg9xfQtq_4av/view?usp=sharing

Here was the text I used
> Hi Kim,
> 
> Of course, happy to connect. Should I reach out to you?
> 
> Thanks,
> Charlotte

**Comment by evertsd** [comment]
Yeah, unfortunately when we highlight tips, we're actually re-creating the DOM within the "textarea" (it's not an actual text area, rather cleverly formatted HTML to give the appearance of a textarea).

Upon re-creating the DOM we have to figure out where to place the cursor, which has flaws at line breaks. Will see what I can do to fix these issues
